<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewProjectDialog</class>
 <widget class="QDialog" name="NewProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>游戏名：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>尺寸：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="screenSize">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>30</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>800 X 600 (4:3)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024 X 768 (4:3)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280 X 960 (4:3)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>854 X 480 (16:9)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1024 X 576 (16:9)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1280 X 720 (16:9)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1366 X 768 (16:9)</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1920 X 1080 (16:9)</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="gameName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>工程目录：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="projectPath">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>339</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPathButton">
   <property name="geometry">
    <rect>
     <x>351</x>
     <y>79</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
